app/functoria_app.ml

```diff
@@ -579,14 +573,15 @@ module Make (P: S) = struct
     let configure (jobs, info) = configure info jobs
     let clean (jobs, info) = clean info jobs
     let build (_jobs, info) = build info
-    let describe (_jobs, info) context t = describe info ~context t
+    let describe (jobs, info) = describe info jobs
 
-    let eval context t =
-      let info = Config.eval context t in
+    let eval ~partial context t =
+      let info = Config.eval ~partial context t in
       let context = Key.context ~stage:`Configure (Key.deps info) in
       let f map =
-        show_keys map @@ Key.deps info;
-        Key.eval map info @@ map
+        let e = Key.eval map info @@ map in
+        Log.info "@[<v>%a@]" (Info.pp false) (snd e) ;
```

Comment:
This prints info about the project (regardless which command is used). You added verbosity info so you can put that in there.